backend/api/server/failure_rotate.go

```diff
@@ -26,9 +31,13 @@ func (s *Server) UpdateFailuresByTeam() {
 			// }
 
 			err = s.cfg.Storage.CreateFailure(failureV1Alpha1.Failure{
-				JiraKey:      failure.JiraKey,
-				JiraStatus:   jiraStatus,
-				ErrorMessage: failure.ErrorMessage,
+				JiraKey:       failure.JiraKey,
+				TitleFromJira: jiraBug.Summary,
+				JiraStatus:    jiraStatus,
```

Comment:
Since now we are getting the jira bug, we can delete `GetJiraStatus` call

```suggestion
JiraStatus: jiraBug.Status,
```